src/test/java/core/basesyntax/service/impl/WriterServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterServiceTest {
+
+    private static final String NOT_EXISTED_BEFORE_WRITING_FILE_NAME =
+            "src/test/resources/notExistedFile.csv";
+    private static final String FILE_TO_WRITE_NAME = "src/test/resources/report.csv";
+    private static WriterService writerService;
+
+    @BeforeClass
+    public static void readerServiceInitialization() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void writeToNullFile_notOk() {
+        String record = "fruit,quantity" + System.lineSeparator()
+                + "banana,100" + System.lineSeparator()
+                + "apple,10";
+        writerService.write(null, record);
```

Comment:
you can also create a method that will read files and match the content of files with the expected.